tokio/src/io/util/async_write_ext.rs

```diff
@@ -186,6 +187,76 @@ cfg_io_util! {
             write_vectored(self, bufs)
         }
 
+        /// Like [`write_all`], except that it writes from a slice of buffers.
+        ///
+        /// Equivalent to:
+        ///
+        /// ```ignore
+        /// async fn write_all_vectored(
+        ///     &mut self,
+        ///     bufs: &mut [IoSlice<'_>],
+        /// ) -> io::Result<()> {
+        ///     let mut pos = 0;
```

Comment:
The `pos` variable is never used. Maybe return it as a result ?

Reply:
My bad, it was left over from a previous iteration. If you have a look at this playground link (ignore the generic param, just for testing) I think this is a valid representation if you were translating to async https://play.rust-lang.org/?version=stable&mode=debug&edition=2024&gist=f25ce7e316683307b91a474f6c7f9496